phylogenetic/defaults/description.md

```diff
@@ -11,10 +11,12 @@ We maintain two views of Lassa virus evolution: one showing evolution of the [L
 
 ### Lineage Calling - Nextclade
 
-A GPC-based Nextclade dataset for calling Lassa lineages is available at the following links:
+Nextclade datasets for lineage assignment are available at the following links:
 
-* [View the Lassa Nextclade scaffold tree](https://next.nextstrain.org/staging/lassa/nextclade8)
-* [Query the Lassa Nextclade dataset](https://clades.nextstrain.org/?dataset-json-url=https%3A%2F%2Fnext.nextstrain.org%2Fstaging%2Flassa%2Fnextclade8)
+| Focus | Reference | Reference Tree | URL to Lassa Nextclade Dataset |
+| :-- | :-- | :-- | :-- |
+| GPC gene | [NC_004296 (Josiah)](https://github.com/nextstrain/lassa/blob/main/nextclade/defaults/gpc/reference.fasta) | [View Tree](https://next.nextstrain.org/fetch/raw.githubusercontent.com/nextstrain/lassa/main/nextclade_data/gpc/tree.json?d=tree&p=full) | [Query GPC Nextclade dataset](https://clades.nextstrain.org/?dataset-url=https://github.com/nextstrain/lassa/tree/main/nextclade_data/gpc/) |
```

Comment:
_non-blocking nit_

Maybe use `HEAD` instead of `main`? The default branch is unlikely to change though so :shrug:

Reply:
Happy to update to `HEAD` (huh, I hadn't used this feature before), basically I want this pointing at the default branch. 

Updated in https://github.com/nextstrain/lassa/commit/ff10c669b74ab08c6a6dae8a6a69ddc46a5cb2e9 and I checked that all the links still work, thanks!